src/apps/logs/domain/notification.py

```diff
@@ -66,6 +67,7 @@ class PublicNotificationLog(_NotificationLogBase, PublicModel):
     notification_descriptions: list
     notification_in_queue: list
     scheduled_notifications: list
+    created_at: datetime.datetime
```

Comment:
Where is this field populated?

Reply:
This field already exists in the database. It just needs to be given to the frontend